src/Parser/Core/Modules/Spells/BFA/AzeriteTraits/MeticulousScheming.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import { calculateAzeriteEffects } from 'common/stats';
+import Analyzer from 'Parser/Core/Analyzer';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'Interface/Others/TraitStatisticBox';
+
+const meticulousSchemingStats = traits => Object.values(traits).reduce((obj, rank) => {
```

Comment:
why construct an object as the return value here? in both cases you use it it is immediately destructured or has the `haste` field used directly.